.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: android
+dist: precise
```

Comment:
Is this change intended?

Reply:
yes, travis stoped working because default image lacks android tools